pkg/scheduler/scheduler.go

```diff
@@ -98,6 +100,7 @@ type schedulerOptions struct {
 	profiles                   []schedulerapi.KubeSchedulerProfile
 	extenders                  []schedulerapi.Extender
 	frameworkCapturer          FrameworkCapturer
+	parallelism                int32
```

Comment:
where is this used?

Reply:
Updated. cmd will call `WithParallelism` for this parameter.